mne/time_frequency/tfr.py

```diff
@@ -878,15 +878,32 @@ def crop(self, tmin=None, tmax=None):
             Start time of selection in seconds.
         tmax : float | None
             End time of selection in seconds.
+        fmin : float | None
+            Lowest frequency of selection in Hz.
+
+            .. versionadded:: 0.18.0
+        fmax : float | None
+            Highest frequency of selection in Hz.
+
+            .. versionadded:: 0.18.0
 
         Returns
         -------
         inst : instance of AverageTFR
             The modified instance.
         """
-        mask = _time_mask(self.times, tmin, tmax, sfreq=self.info['sfreq'])
-        self.times = self.times[mask]
-        self.data = self.data[..., mask]
+        time_mask = _time_mask(self.times, tmin, tmax,
+                               sfreq=self.info['sfreq'])
+
+        self.times = self.times[time_mask]
+        self.data = self.data[..., time_mask]
+
+        freq_mask = _freq_mask(self.freqs, fmin, fmax,
+                               sfreq=self.info['sfreq'])
```

Comment:
you always call _freq_mask with a sfreq that is not None. Why do you think we should support this use case?

Reply:
Well, i thought, for the same purpose you can pass None in ```_time_mask```. To keep the function more flexible. I mean, ```_time_mask``` and ```freq_mask``` work mostly fine if you don't pass the ```sfreq``` argument. Like in```_time_mask```,  ```sfreq``` is only needed if, for example, you want to include ```fmax=50```, but the corresponding frequency in ```freqs``` is something like 50.0001.
But if you say that in all cases in which one might want to use ```_freq_mask```, there will always be an ```sfreq``` to pass, we can certainly remove the None default.

Reply:
@agramfort Would you like to also remove the `sfreq` used in `_time_mask`?

Reply:
If that doesn't hurt the test I'm +1 on removing the None for both: `_time_mask` and `_freq_mask`

Reply:
`git grep "_time_mask("` says that there are places we currently don't use `sfreq` with `_time_mask`, so we should leave support for `sfreq=None` in `_time_mask` for now. We probably should not remove `sfreq=None` support from `_time_mask` without being very careful about the consequences. Hopefully its necessity / use is already covered by a test. But this is far another PR if we want to do it.

In the meantime, @DiGyt path of least resistance is probably to remove support for `None` for `sfreq` in `_freq_mask` here, since we don't currently need it.

Reply:
Alright, ```sfreq=None``` default removed in ```_freq_mask```!